Cargo.toml

```diff
@@ -78,5 +78,5 @@ sys-info = "0.9"
 [target."cfg(any(target_os = \"ios\", target_os = \"macos\"))".dependencies]
 cocoa = "0.24"
 core-graphics = "0.22"
-objc = { version = "0.2", features = [ "exception" ] }
+objc = "0.2"
```

Comment:
I really think you should keep the `exception` as a default feature flag, disabled on docs.rs, this was part of the audit